packages/backend/package.json

```diff
@@ -20,6 +20,15 @@
           "default": 30,
           "maximum": 120,
           "description": "Build timeout (in minutes)"
+        },
+        "bootc.builder": {
+          "type": "string",
+          "default": "Centos",
```

Comment:
should be CentOS throughout code + options

Reply:
Fixed.